config.py

```diff
@@ -13,8 +13,9 @@
 IMPORTING ENVIRONMENT VARIABLES'''
 
 BEAT_WEBSITE_OAUTH_CREDENTIALS = os.environ.get('BEAT_WEBSITE_OAUTH_CREDENTIALS')
-PAYPAL_SANDBOX_CLIENT_ID = os.environ.get('PAYPAL_SANDBOX_CLIENT_ID')
-PAYPAL_SANDBOX_CLIENT_SECRET = os.environ.get('PAYPAL_SANDBOX_CLIENT_SECRET')
+PAYPAL_CLIENT_ID = os.environ.get('PAYPAL_CLIENT_ID')
+PAYPAL_CLIENT_SECRET = os.environ.get('PAYPAL_CLIENT_SECRET')
+PAYPAL_LIVE = (os.environ.get('PAYPAL_LIVE')).lower()
```

Comment:
Added the lower() so there is no risk of this not working because PAYPAL_LIVE was entered as "TRUE" or "True" instead of "true"